server/middlewares/passport.ts

```diff
@@ -18,12 +19,28 @@ export default function createMiddleware(providerName: string) {
 
           if (err.id) {
             const notice = err.id.replace(/_/g, "-");
-            const hasQueryString = err.redirectUrl?.includes("?");
+            const redirectUrl = err.redirectUrl ?? "/";
```

Comment:
I swear there are 10 places in the codebase where `Url` is used instead of `Path` and it's mighty confusing, need to do an audit 🙈